test/Twilio.Test/Http/SystemNetHttpClientTest.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Castle.Core.Logging;
+using NUnit.Framework;
+using Twilio.Http;
+using HttpMethod = Twilio.Http.HttpMethod;
+
+namespace Twilio.Tests.Http
+{
+    public class MockResponseHandler : DelegatingHandler
```

Comment:
Had to add this to mock out the actual http calls in `System.Net.Http.HttpClient`. Dont think we'll use it in other tests so i left it here but we can move it to its own file anyone cares enough.